cmd/root.go

```diff
@@ -57,69 +57,57 @@ func initConfig() {
 	// Check args
 	if len(os.Args) == 1 {
 		return
-	} else if len(os.Args) != 2 {
+	}
+
+	if len(os.Args) == 2 {
 		switch os.Args[1] {
 		case "config":
 			return
 		case "help":
 			return
-		default:
-			initConfigDDKey()
 		}
 	}
+
+	initConfigDDKey()
 }
 
 func initConfigDDKey() {
-	// check file exist
-	var dogrcExist bool
-	_, err := os.Stat(dogrcFile)
+	_, err := os.Stat(dogleashFile)
 	if err != nil {
-		dogrcExist = false
-	} else {
-		dogrcExist = true
+		log.Fatalf("\nDogleash config file does not exist. [%s]\n", dogleashFile)
 	}
-	var dogleashExist bool
-	_, err = os.Stat(dogleashFile)
+
+	viper.SetConfigFile(dogleashFile)
+	err = viper.ReadInConfig()
 	if err != nil {
-		dogleashExist = false
-	} else {
-		dogleashExist = true
+		log.Fatal(err)
 	}
 
-	// set config api/app keys
-	if dogleashExist {
-		viper.SetConfigFile(dogleashFile)
-		err := viper.ReadInConfig()
+	err = viper.Unmarshal(&DC)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if DC.Current == "dogrc" {
+		_, err := os.Stat(dogrcFile)
 		if err != nil {
-			log.Fatal(err)
+			log.Fatal("\nCurrent config is set as \"dogrc\", but ~/.dogrc does not exist.")
 		}
 
-		err = viper.Unmarshal(&DC)
+		dogrc, err := ini.Load(dogrcFile)
 		if err != nil {
 			log.Fatal(err)
 		}
 
-		if DC.Current == "dogrc" {
-			if dogrcExist {
-				dogrc, err := ini.Load(dogrcFile)
-				if err != nil {
-					log.Fatal(err)
-				}
-				viper.SetDefault("api_key", dogrc.Section("Connection").Key("apikey").String())
-				viper.SetDefault("app_key", dogrc.Section("Connection").Key("appkey").String())
-			} else {
-				log.Fatal("\ncurrent config is dogrc. but does not exist ~/.dogrc")
-			}
-		} else {
-			for _, o := range DC.Organizations {
-				if o.Name == DC.Current {
-					viper.SetDefault("api_key", o.APIKey)
-					viper.SetDefault("app_key", o.APPKey)
-				}
+		viper.SetDefault("api_key", dogrc.Section("Connection").Key("apikey").String())
+		viper.SetDefault("app_key", dogrc.Section("Connection").Key("appkey").String())
+	} else {
```

Comment:
How about removing an else statement in terms of the early return?

Reply:
After loading configs from the file, it reads environment variables to overwrite configs.
I think we need to use both if and else statements in this case.